Plugins/Bugsnag/Source/Bugsnag/Private/Apple/ApplePlatformBugsnag.cpp

```diff
@@ -25,9 +25,51 @@
 
 DEFINE_PLATFORM_BUGSNAG(FApplePlatformBugsnag);
 
+#if UE_EDITOR
+
+#import <Bugsnag/BSG_KSCrashReportWriter.h>
+#import <BugsnagPrivate/BugsnagConfiguration+Private.h>
+
+#define IS_PIE_WORLD_KEY "isPlayInEditorWorld"
+#define UNREAL_ENGINE_KEY "unrealEngine"
+
+static void FApplePlatformBugsnag_OnCrashHandler(const BSG_KSCrashReportWriter* writer)
+{
+	writer->beginObject(writer, UNREAL_ENGINE_KEY);
+	writer->addBooleanElement(writer, IS_PIE_WORLD_KEY, GIsPlayInEditorWorld);
```

Comment:
Is it worth writing `GIsEditor` / `GIsBuildMachine` too?

Reply:
No because the crash handler is deliberately only installed `if (GIsEditor || GIsBuildMachine)`.

Reply:
But you might want to know which of those was true?

Reply:
Why?

Reply:
This is now moot due to removing the build machine considerations.  Resolving.